### ISSUE TITLE
Convert tiny-yolov3 to Tflite.

### ISSUE THREAD

**Issue by apoorvopen** [jinyu121/DW2TF#33]
Can you help me in converting the tiny-yolov3 model to .Tflite to run it on android? Please let me know if you have any leads for me.

**Comment by weilanShi** [comment]
> Can you help me in converting the tiny-yolov3 model to .Tflite to run it on android? Please let me know if you have any leads for me.
Hello, have you succeeded in converting. weight to. tflite?

**Comment by weilanShi** [comment]
Hello, I have succeeded in converting. weigths to. tflite. The steps are as follows: first, convert. weights. cfg to. pb and .ckpt, referring specifically to: https://github.com/jinyu121/DW2TF/issues/30 https://github.com/jinyu121/DW2TF/releases/tag/freez/tensorflow tool e_graph, then generate. pb. Finally, the tensorflow tool toco is used to convert to. tflite.

**Comment by nscotto** [comment]
Could you provide a details of the commands you entered to get the tflite version from the darknet weights?
I can't figure out what to enter for freezing the graph, I get errors on every trials, and your links don't work.

Thank you in advance.

**Comment by peace195** [comment]
@apoorvopen I did it here: https://github.com/peace195/tensorflow-lite-yolo-v3

I would I would appreciate if you give me a star for this project 👍

**Comment by nscotto** [comment]
Thanks, I will.
Before I try, does it support different number of channels? (1 for grayscale images)

**Comment by peace195** [comment]
@nscotto Yes, please change the size of your image in this line: https://github.com/peace195/tensorflow-lite-yolo-v3/blob/master/convert_weights_pb.py#L42

**Comment by nscotto** [comment]
Thanks! Star deserved.

**Comment by peace195** [comment]
@nscotto  I need some helps: https://github.com/peace195/tensorflow-lite-yolo-v3/issues/1

**Comment by spoorgholi74** [comment]
> @nscotto I need some helps: [peace195/tensorflow-lite-yolo-v3#1](https://github.com/peace195/tensorflow-lite-yolo-v3/issues/1)

your repo only gets the common yolov3 config right?
so if pruned or change it should be change accordingly?

**Comment by peace195** [comment]
> > @nscotto I need some helps: [peace195/tensorflow-lite-yolo-v3#1](https://github.com/peace195/tensorflow-lite-yolo-v3/issues/1)
> 
> your repo only gets the common yolov3 config right?
> so if pruned or change it should be change accordingly?

Yes. You are correct.

**Comment by peace195** [comment]
@spoorgholi74 You can change the YOLO's structure at here: https://github.com/peace195/tensorflow-lite-yolo-v3/blob/master/yolo_v3.py

**Comment by spoorgholi74** [comment]
@peace195 yeah but it would be better to write a function to read the .cfg file instead of changing the lines

**Comment by peace195** [comment]
@spoorgholi74 I have a version of it. I will push the code when I have time. 👍
